djangosaml2/views.py

```diff
@@ -145,7 +145,10 @@ def login(request,
     #
     # Read more in the official SAML2 specs (3.4.4.1):
     # http://docs.oasis-open.org/security/saml/v2.0/saml-bindings-2.0-os.pdf
-    binding = BINDING_HTTP_POST if conf._sp_authn_requests_signed else BINDING_HTTP_REDIRECT
+    binding = BINDING_HTTP_REDIRECT
+    if hasattr(conf, '_sp_authn_requests_signed') and \
+            conf._sp_authn_requests_signed:
+        binding = BINDING_HTTP_POST
```

Comment:
This 80-chars per line is actually the only PEP8 rule that is officially not respected by Django guidelines, and also by most of other serious projects. The reasoning is quite obvious - this rule comes from the previous century and sounds ridiculous in today's era of huge and wide screens. From what I have observed, 120 columns seems to be more preferred these days, even though there is not a strict limit.

Now, back to the topic :-) Thank for the new commit, and if I can bother you just a bit more - would you mind squashing those two commits into one? Thanks :-)